src/templates/bootstrap.tpl.ts

```diff
@@ -83,7 +83,8 @@ export const FORMIO_BOOTSTRAP: FormioTemplate = {
         currency: getTemplate({ gulp_inject: './bootstrap/components/currency.html' }),
         select: {
             component: {
-                template: { gulp_inject: './bootstrap/components/select.html' }
+                template: { gulp_inject: './bootstrap/components/select.html' },
+                styles: ['.required .control-label::after { content:" *"; color:red; }']
```

Comment:
Since these styles are used over and over, we should either store them as a variable, or apply them to an element that contains the components and have the css target them.